src/main/resources/templates/dev-hub/developerGuidelines.html

```diff
@@ -11,61 +11,99 @@
 <div layout:fragment="content">
   <h1 class="govuk-heading-xl">Developer guidelines</h1>
 
-  <p class="govuk-body-l">To help achieve the successful development of your application, when using
-    the
-    Companies House API you should adhere to a few simple guidelines:</p>
+  <h2 class="govuk-heading-m">Contents</h2>
 
   <ol class="govuk-list govuk-list--number">
-    <li><a href="#resources" class="govuk-link">Data resources</a></li>
-    <li><a href="#security" class="govuk-link">Security</a></li>
-    <li><a href="#keysecurity" class="govuk-link">Securing API keys</a></li>
+    <li><a class="govuk-link" href="#rateLimits">API rate limits</a></li>
+    <li><a class="govuk-link" href="#enums">Enumerated Types</a></li>
+    <li><a class="govuk-link" href="#resources">Data Resources</a></li>
+    <li><a class="govuk-link" href="#appSecurity">Application Security</a></li>
+    <li><a class="govuk-link" href="#keySecurity">API Key Security</a></li>
   </ol>
 
+  <section id="rateLimits">
+    <h2 class="govuk-heading-m">API rate limits</h2>
+    <p class="govuk-body">Rate limiting is applied to the Companies House API to ensure a
+      high-quality service is delivered for all users, and to protect client applications from
+      unexpected loops.</p>
+    <p class="govuk-body">
+      You can make up to 600 requests within a five-minute period. If you exceed this limit, you
+      will receive a <code>429 Too Many Requests</code> HTTP status code for each request made
+      within the
+      remainder of the five-minute window. At the end of the period, your rate limit will reset back
+      to its maximum value of 600 requests.
+    </p>
+    <p class="govuk-body">
+      If you have an application that requires a higher rate limit than this default, <a
+        th:href="@{{chsUrl}/help/contact-us(chsUrl=${@environment.getProperty('chs.url')})}">contact
```

Comment:
Does this `chsUrl `go to the main domain service with a relevant contact page?

Reply:
This was copied from the footer bar which currently works in live.